Additional manpage conversions

frankcrawford opened the pull request:
Updated rdiff-backup.1.adoc
Added rdiff-backup-statistics.1.adoc

ericzolf commented on docs/rdiff-backup-statistics.1.adoc:
```diff
@@ -0,0 +1,86 @@
+making the given repository.
+
+--quiet::
+Suppress printing of the '[.code]``Processing statistics from session...``"
```
the code section starts with a single quote and ends with a double one.

And I think, it would be nicer to have three backquotes instead of the `[.code]` thingy, i.e.

```asciidoc
'```bla bla```'
```

It would be more readable as raw code, and it would even give us curved quotes, which look slightly nicer, see https://docs.asciidoctor.org/asciidoc/latest/text/quotation-marks-and-apostrophes/

This last comment is of course valid for the whole document. Else everything looks good to me.

frankcrawford commented:
Updated.

The reason why it is use [.code] all through the file is that it had the most consistent result in the rdiff-backup.1.adoc file.  During the discussion of globbing and pattern matching, there were lots of issue, that required random backslashes, etc, to resolve.  I converted to using them almost exclusively, and just did it through out the files.

ericzolf commented on docs/rdiff-backup.1.adoc:
```diff
@@ -333,11 +331,11 @@ The default is to include other filesystems.
 --min-file-size _sizeinbytes_:: Exclude files that are smaller than the given size in bytes.
 
-[[statistics-options]]
+[[satistics-options]]
```
Sorry,  I see a missing t here.

frankcrawford replied:
Done

frankcrawford replied:
Something broke in the testing, it can't find setuptools-scm.

ericzolf replied:
There were CDN issues during the day,  just amend the last commit and force push it.

ericzolf replied:
Sorry, there is a re run jobs button in the checks tab, no need to force push!
